data/src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java

```diff
@@ -461,8 +461,7 @@ private String generateCommandPayload(String database, String command, ClientReq
     }
 
     private void addCommandHeaders(Map<String, String> headers) {
-        headers.put(HttpHeaders.CONTENT_TYPE, "application/json");
-        headers.put("Fed", "True");
```

Comment:
Am I mistaken that this is to indicate "AAD federated authentication", but authentication should be dependent on the KCSB?